static/js/giphy.js

```diff
@@ -0,0 +1,81 @@
+import {renderGrid} from "@giphy/js-components";
+import {GiphyFetch} from "@giphy/js-fetch-api";
+import $ from "jquery";
+import {throttle} from "throttle-debounce";
+
+import render_giphy_picker from "../templates/giphy_picker.hbs";
+
+import * as compose_ui from "./compose_ui";
+
+const gf = new GiphyFetch(page_params.giphy_api_key);
+let search_term = "";
+
+function fetchGifs(offset) {
+    const config = {
+        offset,
+        limit: 25,
+    };
+    if (search_term === "") {
+        // Get the trending gifs by default.
+        return gf.trending(config);
+    }
```

Comment:
I kinda wonder if this is a good idea -- is this what other team chat products do?  It seems really unlikely to me that these would be useful as a reaction to a specific thing.

Reply:
Mattermost shows trending GIFs by default. Some show recently used GIFs, for that we will have to track every user separately. Using their original email for that could be good idea. This is something we can iterate on later.

Reply:
Generally, I would not copy Mattermost's behavior if we don't think it makes sense; I haven't been impressed with their UX design.  

I removed mention of this detail from the /help/ docs, since it's not important there.  But I'm fine with making this call after seeing it on chat.zulip.org.